README.md

```diff
@@ -13,6 +13,26 @@ The following describes how to configure Datadog to gather metrics from Artifact
 | 0.2.0   | 7.7.3       | 3.8.0 | N/A          | N/A             | N/A       |
 | 0.1.1   | 7.6.3       | 3.6.2 | N/A          | N/A             | N/A       |
 
+## Table of Contents
+
+1. [Environment Configuration](#environment-configuration)
+2. [Fluentd Configuration for Datadog](#fluentd-configuration-for-datadog)
+    * [Configuration steps for Artifactory](#configuration-steps-for-artifactory)
+    * [Configuration steps for Xray](#configuration-steps-for-xray)
+    * [Configuration steps for Mission Control](#configuration-steps-for-mission-control)
+    * [Configuration steps for Distribution](#configuration-steps-for-distribution)
+    * [Configuration steps for Pipelines](#configuration-steps-for-pipelines)
+3. [Fluentd Installation](#fluentd-installation)
+    * [OS / Virtual Machine](#os--virtual-machine)
+    * [Docker](#docker)
+    * [Kubernetes Deployment with Helm](#kubernetes-deployment-with-helm)
+    * [Kubernetes Deployment without Helm](#kubernetes-deployment-without-helm)
+4. [Datadog Setup](#datadog-setup)
+    * [Dashboards](#dashboards)
+5. [Demo Requirements](#demo-requirements)
+6. [Generating data for Testing](#generating-data-for-testing)
+7. [References](#references)
+
 ## Environment Configuration
```

Comment:
This section should read as follows:

We rely heavily on environment variables so that the correct log files are streamed to your observalibity dashboards. Ensure that you have the exported the JF_PRODUCT_DATA_INTERNAL environment variable to the correct path for your product. See your specific product installation below to find the correct path.

Reply:
We should probably create a pre-requisites section which includes the following:

Pre-Requisites:
Ensure that you have a standard installation of OS without customizations
Ensure that you have sudo access 
Ensure that you have the following tools available
- a file editor available on your VM - vi, nano, pico or similar
- wget/curl so that you can download files from the interner
- access from the VM to the internet

Then have the environment variables section

Reply:
Prerequisites are not required since
sudo access is not a requirement for installations
Also, a file editor is not needed. They can edit the file locally and copy it to the required location

Reply:
Why are these prereq's?

Ensure that you have a standard installation of OS without customizations
Our log analytics solution should work regardless of what they have done to the OS until they blocked egress through FW.

Ensure that you have sudo access
Not necessary they can use the zip installer to do a user based install.